src/scripts/fix_variant_descriptions.php

```diff
@@ -941,7 +941,7 @@ function ($sVOT)
                                     } elseif ($aVOT['RNA'] == str_replace('?', '', $aVVVot['data']['RNA'])
                                         || (strpos($aVOT['RNA'], 'spl') !== false && preg_match('/[0-9]+[+-][0-9]+/', $aVVVot['data']['DNA']))) {
                                         // We ignore small differences, where maybe the RNA has been verified.
-                                    } elseif (lovd_getVariantInfo(lovd_fixHGVS('c' . strstr($aVOT['RNA'], '.'), 'c'), '', true)
+                                    } elseif (lovd_isHGVS(lovd_fixHGVS('c' . strstr($aVOT['RNA'], '.'), 'c'))
```

Comment:
This may not be correct; `''` is replaced by `false` here, which is something different.

Reply:
Yes, I discussed this with you when I changed this code: the `''` was incorrect, and it should have been `false` in the original code.

Reply:
You're right, I forgot about this and rushed to a conclusion in an attempt to finish my review before catching the train :sweat_smile: Sorry!